test/d2l-offscreen.html

```diff
@@ -21,17 +22,18 @@
 			</template>
 		</test-fixture>
 
-		<script>
-			suite('d2l-offscreen', function() {
-				setup(function() {
-					this.fixture = fixture('basic');
-					this.element = this.fixture.querySelector('d2l-offscreen');
-				});
+		<script type="module">
+import '../d2l-offscreen.js';
+suite('d2l-offscreen', function() {
+	setup(function() {
+		this.fixture = fixture('basic');
```

Comment:
Now complaining about the `this`: `error  Unexpected 'this'  no-invalid-this`

Reply:
Maybe disable the "no-invalid-this" check in the test directory? This could be done before conversion.